src/Converters.ts

```diff
@@ -13,13 +13,33 @@ export function milliSecondsToText(ms: number) {
 }
 
 let distanceFormat: Intl.NumberFormat = new Intl.NumberFormat('en', { maximumFractionDigits: 1 })
-export function setDistanceFormat(_distanceFormat: Intl.NumberFormat) {
-    distanceFormat = _distanceFormat
+let distanceFormat2 = new Intl.NumberFormat('en', { maximumFractionDigits: 2 })
+
+export function initDistanceFormat(lang: string) {
+    distanceFormat = new Intl.NumberFormat(lang, { maximumFractionDigits: 1 })
+    distanceFormat2 = new Intl.NumberFormat(lang, { maximumFractionDigits: 2 })
+}
+
+export function kmToMPHIfMiles(value: number, showDistanceInMiles: boolean, roundTo10 = false) {
+    return showDistanceInMiles
+        ? roundTo10
+            ? Math.round(value / 1.60934 / 10.0) * 10
+            : Math.round(value / 1.60934)
+        : Math.round(value)
+}
+
+export function meterToFt(value: number) {
+    return value / 0.3048
```

Comment:
@karussell I have a doubt, the good formula is rather :  meterToFt => value * 3,28084 ?
same for meterToMiles (value * 1609.34) or I'm wrong ?